crates/build/re_types_builder/src/codegen/rust/blueprint_validation.rs

```diff
@@ -18,7 +18,8 @@ pub(crate) fn generate_blueprint_validation(
 ) {
     let blueprint_scope = Some("blueprint".to_owned());
     let mut code = String::new();
-    code.push_str(&format!("// {}\n", autogen_warning!()));
+    code.push_str(&format!("// {}\n\n", autogen_warning!()));
+    code.push_str("#![expect(clippy::empty_line_after_doc_comments)]\n\n");
```

Comment:
a bit strange, why not fix that empty line?

Reply:
Because I suspect that's gonna be a lot harder

Reply:
kk, didn't check; may or may not 😄